data_safe_haven/infrastructure/programs/sre/software_repositories.py

```diff
@@ -161,7 +161,7 @@ def __init__(
                 container_group_name=f"{stack_name}-container-group-software-repositories",
                 containers=[
                     containerinstance.ContainerArgs(
-                        image="caddy:2.8.4",
+                        image="caddy:2.8",
```

Comment:
Why has this changed?

Reply:
Feels like an over specification to me.

Reply:
No, we need it. The automated Docker image updater looks for the latest release, not the latest minor release, so this change is made automatically.